packages/devtools_app/test/screens/inspector_v2/inspector_integration_test.dart

```diff
@@ -2,17 +2,21 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:io';
```

Comment:
you may need to add `@TestOn('vm')` since this is using dart:io and won't be able to run on the web